src/receivers/SocketModeReceiver.ts

```diff
@@ -148,6 +148,8 @@ export default class SocketModeReceiver implements Receiver {
 
   public start(): Promise<void | WebAPICallResult> {
     // start socket mode client
+    // TODO: We can update the returned type from WebAPICallResult to AppsConnectionsOpenResponse
```

Comment:
I'm fine to hold off merging this PR until this TODO fix if others think we should do so.

Reply:
I don't think we need to hold off on merging the PR. What's blocking us from updating the returned type to `AppsConnectionsOpenResponse`?

Reply:
I will update this part after releasing `@slack/socket-mode@1.1.0` - https://github.com/slackapi/node-slack-sdk/pull/1239